frontend/packages/data-portal/app/utils/featureFlags.ts

```diff
@@ -4,11 +4,11 @@ import { useEnvironment } from 'app/context/Environment.context'
 
 export type FeatureFlagEnvironment = typeof process.env.ENV
 
-export const FEATURE_FLAGS = {
-  methodType: 'dev',
-} as const
+export type FeatureFlagKey = 'methodType'
 
-export type FeatureFlagKey = keyof typeof FEATURE_FLAGS
+export const FEATURE_FLAGS: Record<FeatureFlagKey, FeatureFlagEnvironment[]> = {
+  methodType: ['local', 'dev'],
```

Comment:
Not a blocker: For local environments, should we always default to true or have an env override so that we don't accidentally update the configuration from local testing? 

We can always catch this in the code reviews, so it is not a blocker.